smart-common/src/main/java/org/smartdata/conf/SmartConfKeys.java

```diff
@@ -23,7 +23,6 @@
 public class SmartConfKeys {
   public final static String SMART_DFS_ENABLED = "smart.dfs.enabled";
   public final static boolean SMART_DFS_ENABLED_DEFAULT = true;
-  public final static String SMART_CONF_DIR = "smart.conf.dir";
```

Comment:
yes, i see. In fact, it can not be configured through edit xml files (the script will overwrite its value to default SMART_HOME/conf if you do it in this way), the only way that can set this value is through environment variable. In all cases, SmartServer will get the correct path of the effective config directory, so what i mean was to keep the variable to provide a way to get the real configure path, as it may also have some issues if you have no way to know its configuration directory. For example, adding some other configs in the sub-directory under SMART_CONF_DIR, then how to find these files.